.classpath

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src/main/java"/>
-	<classpathentry kind="src" path="src/main/resources"/>
 	<classpathentry kind="src" path="src/test/server/java"/>
-	<classpathentry kind="src" path="src/test/server/resources"/>
 	<classpathentry kind="src" path="src/test/client/java"/>
-	<classpathentry kind="src" path="src/test/client/resources"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
```

Comment:
Why is this JRE container in the classpath? More precisely: why is there a need to start a JRE container for this project?